packages/text-bitmap/src/formats/XMLTextFormat.ts

```diff
@@ -0,0 +1,39 @@
+import { BitmapFontData } from '../BitmapFontData';
+import { XMLFormat } from './XMLFormat';
+
+/**
+ * BitmapFont format that's XML-based.
+ *
+ * @class
+ * @private
+ */
+export class XMLTextFormat
+{
+    /**
+     * Check if resource refers to text xml font data.
+     *
+     * @static
+     * @private
+     * @param {any} data
+     * @return {boolean} True if resource could be treated as font data, false otherwise.
+     */
+    static test(data: unknown): boolean
+    {
+        return typeof data === 'string' && data.indexOf('<font>\n  <info face=') === 0;
```

Comment:
I think the check here for `data.indexOf` is a little to strict and makes some assumptions about indenting, formatting etc. Not sure it's safe to assume this.Would be in favor of a RegExp instead or maybe something looser like `data.indexOf('<font>') > -1`

Reply:
Replaced with RegExp. Also added test for loading `fnt` based on text format, XML we already have.